### PR TITLE
Fixed up hacky arm and science commands, added xbox controller

### PR DESCRIPTION
**Pull request by azumnanji**

### CODE REVIEW

File: xbox_controller/launch/xbox_controller.launch [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<launch>
+	<!-- joy node -->
+		<node respawn="true" pkg="joy"
+			type="joy_node" name="turtle_joy">
+			<param name="dev" type="string" value="/dev/input/js0"/>
+			<param name="deadzone" value="0.2"/>
```
Comment (wmmc88) [inline xbox_controller/launch/xbox_controller.launch:6]: A lil high

File: science_node/src/science_controller.cpp [hunk 2]
```diff
@@ -1,6 +1,6 @@
 
 #include "ros/ros.h"
-#include "std_msgs/Float32.h"
+#include "std_msgs/Float32MultiArray.h"
```
Comment (wmmc88) [inline science_node/src/science_controller.cpp:3]: Curious: any reason u made this change?)

Comment (azumnanji) [reply science_node/src/science_controller.cpp:3]: the way xbox is written makes it easy to send commands this way. i also thought it was cleaner so I didnt have to have 4 publishers

File: xbox_controller/launch/xbox_controller.launch (hunk 1 above)
Comment (azumnanji) [reply xbox_controller/launch/xbox_controller.launch:6]: put it at 0.1 or maybe lower? leaving at 0.1 for now then

**Pull request merged**